client/src/components/twitter/TwitterLink.js

```diff
@@ -1,68 +1,77 @@
 /**
  * The TwitterLink component displays a "dynamic" button that a user will click to link/unlink their twitter feed.
  */
-import React, { useState } from 'react';
+import { Formik } from 'formik';
+import * as yup from 'yup';
+import React from 'react';
 import { useDispatch, useSelector } from 'react-redux';
-import Button from 'react-bootstrap/Button';
+import { useState } from 'react';
 
 import { linkTwitter } from '../../actions/team';
-
+import { Form, Button, Spinner, Jumbotron } from 'react-bootstrap';
 import './TwitterLink.css';
-import Jumbotron from 'react-bootstrap/Jumbotron';
-import Form from 'react-bootstrap/Form';
-import Col from 'react-bootstrap/Col';
 
 const TwitterLink = () => {
   const dispatch = useDispatch();
-  const [handle, setHandle] = useState('');
-  const [revealInput, setRevealInput] = useState(false);
   const teamId = useSelector((state) => state.team.teamId);
+  const [loadingState, setLoadingState] = useState(false);
```

Comment:
nit: prefer calling this just `isLoading`, or equivalent; i.e. remove the `state` suffix as its implied.